app/src/main/java/org/jellyfin/androidtv/details/ItemPosterPresenter.kt

```diff
@@ -42,6 +44,9 @@ class ItemPosterPresenter(private val context: Context) : Presenter(), IItemClic
 			isFocusableInTouchMode = true
 			setMainImageDimensions(100.dp, 150.dp)
 			mainImageDrawable = TvApp.getApplication().getDrawableCompat(R.drawable.tile_port_video)
+
+			if (baseItem is PlayableItem)
+				setBadge(WatchedBadge(context, baseItem), MultiBadgeImageCardView.BadgeLocation.TOP_RIGHT)
```

Comment:
Might be a bit late but i'd swap the parameter so you specify the location first and the content second

Reply:
I think the current order is more "readable", like set badge watched badge to the top right, but that isn't a strong preference.

Reply:
It's funny because I think it's more readable if you say "At the top right, I want to show the watched badge". It's not a big issue so you can leave it.